### PR TITLE
[major] Add azure pipeline for forked project

### PR DESCRIPTION
**Pull request by awilmore**
I tried to leave the existing build automation as is as much as possible. I did have to update the version of the base alpine image, however, as it was too old to build.

The pipeline will work once this PR is merged: https://github.com/mx51/tf-implementation-shared/pull/323

Here's the pipeline: https://dev.azure.com/mx51/msp-infra/_build?definitionId=299&_a=summary

When this PR is merged to master, we can merge it out onto @nikskiz 's branch: https://github.com/mx51/eks-rolling-update/pull/3

We may need to run a sync with the `master` branch from the parent repo at some point.

NOTE: this code is public!

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -9,7 +9,7 @@ RUN apk add --no-cache curl make && \
 COPY . .
 RUN make dist version=${VERSION}
 
-FROM python:3-alpine3.10
+FROM python:3-alpine3.14
```
Comment (aquam8) [inline Dockerfile:12]: There is a 3.15 version available..

File: azure-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+    - script: make setup && . venv/bin/activate
+      displayName: "Create venv and install packages"
+
+    - script: |
+        VERSION=$( ./determine-version.sh )
+        echo make docker-dist version=$VERSION
+        make docker-dist version=$VERSION
+      displayName: "Build docker image"
+
```
Comment (aquam8) [inline azure-pipelines.yml:39]: Could you please add Twistlock scan if that's possible, thanks!

File: Dockerfile (hunk 1 above)
Comment (awilmore) [reply Dockerfile:12]: Yep, I realised after I pushed. Will fix.

File: azure-pipelines.yml (hunk 2 above)
Comment (awilmore) [reply azure-pipelines.yml:39]: Will do.

**Review by aquam8 — APPROVED** [review]
Awesome! Let me know if you need credentials to be added to the pipeline library for sysdig

**Pull request merged**